webpack.config.prod-dev.js

```diff
@@ -46,14 +46,6 @@ module.exports = env => {
             origin: proxyTarget
           },
         },
-        '/workspace-loader': {
```

Comment:
Is this a related change to the PR?

Reply:
No. This is not a related change to the PR. I will remove it.